app/src/main/java/io/pumpkinz/pumpkinreader/data/NewsAdapter.java

```diff
@@ -74,6 +80,16 @@ public void onBindViewHolder(RecyclerView.ViewHolder viewHolder, int position) {
                 News news = this.dataset.get(position);
                 NewsViewHolder newsViewHolder = (NewsViewHolder) viewHolder;
 
+                int color;
+
+                if (PreferencesUtil.isNewsRead(fragment.getActivity(), news)) {
+                    color = fragment.getResources().getColor(R.color.grey_200);
+                } else {
+                    color = fragment.getResources().getColor(R.color.grey_1000w);
```

Comment:
wow, I didn't know that grey_1000w is actually white until I saw it on the Android Studio.